LayoutTests/accessibility/ios-simulator/date-picker-focus-notification.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../../resources/js-test.js"></script>
+<script src="../../resources/accessibility-helper.js"></script>
+<script src="../../resources/ui-helper.js"></script>
+</head>
+<body>
+
+<input id="datefield" type="date">
+
+
+<script>
+var output = "this tests that when a date field shows a popover, no focused event is fired.\n\n";
+
+function focusCallback(notification) {
+    if (notification === "AXFocusChanged") {
+    	output += "Received AXFocusChanged event\n";
+    }
+}
+
+if (window.accessibilityController) {
+	window.jsTestIsAsync = true;
+	var datefield = accessibilityController.accessibleElementById("datefield");
+	datefield.addNotificationListener(focusCallback);
+	
+	setTimeout(async function() {
+		await UIHelper.activateElement(document.getElementById('datefield'));
```

Comment:
For better or for worse, we have landed on using double quotes for strings in our tests